js/src/common/components/ModalManager.js

```diff
@@ -43,8 +43,6 @@ export default class ModalManager extends Component {
     this.showing = true;
     this.component = component;
 
-    if (app.current) app.current.retain = true;
```

Comment:
We just assume this was a performance optimization, right?

Reply:
Yeah I guess... This isn't available in Mithril 2 anyway, and removing it didn't seem to change anything.